__tests__/app.test.js

```diff
@@ -201,9 +199,30 @@ describe("DELETE /api/comments/:comment_id", () => {
         expect(msg).toBe('bad request')
       })
   });
-
 });
 
+describe('/api/users', () => {
+  test('GET 200: Responds with all users', () => {
+  return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({ body }) => {
+      const { users } = body;
+      expect(Array.isArray(users)).toBe(true);
+      expect(users).toHaveLength(4);
+      users.forEach(
+          ({ username,
+            name,
+            avatar_url }) => {
+            expect(typeof username).toBe("string");
+            expect(typeof name).toBe("string");
+            expect(typeof avatar_url).toBe("string");
+          }
+        );
```

Comment:
This is a great test!
You could cut line 211 ✂️ if you wanted, because you are checking the length and using a `forEach` which means it must be an array, which makes line 211 unnecessary ( but it does no harm either )